v6/scripts/checklockconflicts.sh

```
@@ -0,0 +1,12 @@
+if [ -f ./package-lock.json ]; then
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)